pkg/middleware/middleware.go

```diff
@@ -66,6 +67,12 @@ func (m *JWTMiddleware) HandlerFunc() gin.HandlerFunc {
 			m.respondWithError(c, http.StatusUnauthorized, err.Error())
 			return
 		}
+
+		if token.UserID == "" || token.AccountID == "" {
+			logrus.Infof("Missing essential claims from token - [user_id:%s][account_id:%s] for user [%s], sub [%s]",
```

Comment:
Can you use our logger instead? Same as you use in signup_service.go

Reply:
no problem, fixed